src/Edelstein.Service.Game/Commands/Handling/LevelUpCommand.cs

```diff
@@ -11,7 +11,7 @@ public class LevelUpCommand : AbstractCommand<LevelUpCommandOption>
 
         public LevelUpCommand(Parser parser) : base(parser)
         {
-            Aliases.Add("LevelUp");
+            Aliases.Add("LvlUp");
             Aliases.Add("LVLUP");
```

Comment:
Looks good, maybe remove the 2nd LVLUP? As when the capitalization works again it would be redundant